services/src/main/scala/cromwell/services/metadata/CallMetadataKeys.scala

```diff
@@ -16,6 +16,7 @@ object CallMetadataKeys {
   val Failures = "failures"
   val Stdout = "stdout"
   val Stderr = "stderr"
+  val TaskLog = "task.log"
```

Comment:
Personally I was never a huge fan of dynamic task log names, but others may be...